input_rpc.go

```diff
@@ -1,6 +1,7 @@
 package kvm
```

Comment:
I think this is wholly unrelated to this PR (while cool), and conflicts with some of @ym's WIP. Can we not include these changes in this PR?

Reply:
sure. these changes were part of the process of reducing interference with the kvm system, before actually figuring out the root cause of latency and perceived interference were actually the mismatched RTP timestamps (which I've solved by moving audio tracks into their own MediaStream)

Reply:
Removed